google/cloud/spanner/database_admin_connection.cc

```diff
@@ -620,23 +624,25 @@ class DatabaseAdminConnectionImpl : public DatabaseAdminConnection {
   std::unique_ptr<RetryPolicy const> retry_policy_prototype_;
   std::unique_ptr<BackoffPolicy const> backoff_policy_prototype_;
   std::unique_ptr<PollingPolicy const> polling_policy_prototype_;
+  std::unique_ptr<BackgroundThreads> background_threads_;
```

Comment:
Consider a comment here, objects of this class (most of the time) creates a pool of threads, and joins those threads in its destructor.  Those threads should not gain ownership of the object (via a `std::shared_ptr<>`) or deadlocks ensue during shutdown

Reply:
Done.